README.md

```diff
@@ -18,6 +18,95 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #XX - Fat Arrow Functions
+> YYYY-MM-DD by [@pklinger](https://github.com/pklinger/)
+
+Inroduced as a new feature in ES6, fat arrow functions may come as a handy tool to write more code in less lines. The name comes from its syntax as `=>` is a 'fat arrow' compared to a thin arrow `->`. Some programmers might already know this type of functions from different languages such as Haskell as 'lambda expressions' respectively 'anonymous functions'. It is called anonymous, as these arrow functions do not have a descriptive function name.
+
+### What are the benefits?
+* Syntax: less LOC; no more typing `function` keyword over and over again
+* Semantics: capturing the keyword `this` from the surrounding context
+
+### Simple syntax example
+Have a look at these two code snippets, which exactly do the same job. You will quickly understand what fat arrow functions do.
+
+```javascript
+// general syntax for fat arrow functions
+(param) => expression
+
+
+// using functions
+var arr = [5,3,2,9,1];
+let arrFunc = arr.map(function(x) {
+  return x * x;
+});
+console.log(arr)
+
+// using fat arrow
+var arr = [5,3,2,9,1];
+let arrFunc = arr.map((x) => x*x);
+console.log(arr)
+```
+
+As you may see, the fat arrow function in this case may save you time typing out the parentheses as well as the function and return keywords. I would advice you to always write parentheses around the parameter inputs as the parentheses will be needed for multiple input parameters such as in `(x,y) => x+y` anyways. It is just a way to cope with forgetting them in different use cases. But the code above would also work like this: `x => x*x`. So far these are only syntactical improvements, which lead to less LOC and better readability. 
```

Comment:
Yes, it is. I wrote comments while reading, so see the note after I wrote first comment regarding parentheses :smile_cat: